bin/highlight.ts

```diff
@@ -77,8 +77,10 @@ async function markdown(file: string): Promise<void> {
 
     let html = highlight(inner, lang);
 
+    html = html.replace(/[\r\n]/g, '').replace(/\{\{\</g, '{\\{<');
```

Comment:
> I guess we need to look at replacing the blank lines with <br> in the highlight script too

```suggestion
    html = html.replace(/[\r\n]/g, '<br>').replace(/\{\{\</g, '{\\{<');
```

This *seems* to work?

<img width="656" alt="image" src="https://user-images.githubusercontent.com/94662631/191975365-59076ae0-7894-44b5-a21a-25a86afa6d01.png">

<img width="688" alt="image" src="https://user-images.githubusercontent.com/94662631/191975473-d6ef84bc-e80a-4dcc-8304-b45d120364b9.png">

Reply:
> However, the "Copy to clipboard" button in every code block doubles every empty line -- was this already happening?

I haven't been able to replicate this on production or on my development server (w/ the `<br>` change) 🤔